rotkehlchen/api/v1/schemas.py

```diff
@@ -720,47 +720,89 @@ def generate_fields_post_validation(self, data: dict[str, Any]) -> dict[str, Any
         return {'group_by_event_ids': data['group_by_event_ids']}
 
 
-class EvmEventSchema(Schema):
+class CreateHistoryEventSchema(Schema):
     """Schema used when adding a new event in the EVM transactions view"""
-    tx_hash = EVMTransactionHashField(required=True)
-    sequence_index = fields.Integer(required=True)
     # Timestamp coming in from the API is in seconds, in contrast to what we save in the struct
     timestamp = TimestampField(ts_multiplier=1000, required=True)
-    location = LocationField(required=True)
-    event_type = SerializableEnumField(enum_class=HistoryEventType, required=True)
-    asset = AssetField(required=True, expected_type=Asset, form_with_incomplete_data=True)
     balance = fields.Nested(BalanceSchema, required=True)
+    entry_type = SerializableEnumField(enum_class=HistoryBaseEntryType, required=True)
+    sequence_index = fields.Integer(required=False)
+    location = LocationField(required=False)
+    event_type = SerializableEnumField(enum_class=HistoryEventType, required=False)
+    asset = AssetField(required=False, expected_type=Asset, form_with_incomplete_data=True)
     location_label = fields.String(required=False)
     notes = fields.String(required=False)
     event_subtype = SerializableEnumField(
         enum_class=HistoryEventSubType,
-        required=False,
-        load_default=HistoryEventSubType.NONE,
         allow_none=True,
     )
-    counterparty = fields.String(load_default=None)
-    product = SerializableEnumField(enum_class=EvmProduct, load_default=None)
-    address = EvmAddressField(load_default=None)
+    counterparty = fields.String(required=False, allow_none=True)
+    product = SerializableEnumField(enum_class=EvmProduct, required=False, allow_none=True)
+    address = EvmAddressField(required=False, allow_none=True)
+    event_identifier = fields.String(required=False)
+    fee_recipient = EvmAddressField(required=False)
```

Comment:
This is not required. This is the location label

```suggestion
    fee_recipient = EvmAddressField(required=False)
```